packages/react-native-session-replay/android/src/main/kotlin/com/datadog/reactnative/sessionreplay/ColorUtils.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.reactnative.sessionreplay
+
+import androidx.annotation.VisibleForTesting
+
+private const val HEX_COLOR_INCLUDING_ALPHA_LENGTH: Int = 8
+
+internal fun resolveBackgroundColorAsHex(backgroundColor: Int): String {
```

Comment:
it is not clear which format will we return. Normally everyone can expect ARGB, but this method returns RGBA. Maybe it makes sense to rename this method to `formatAsRgba` or something similar? Also obviously no need to mention `background` in the name, this method works with any color.

Reply:
👍 I'll rename it to formatAsRgba